packages/core/src/card/Card.css

```diff
@@ -6,6 +6,11 @@
   border-style: var(--salt-container-borderStyle);
   border-color: var(--salt-container-primary-borderColor);
   padding: var(--saltCard-padding, var(--salt-spacing-300));
+
+  font-family: var(--salt-text-fontFamily);
```

Comment:
If utik components were placed inside the Card would they then inherit the font-family etc from the Card instead of body and therefore still have the compatibility issues?

Reply:
Good question, I checked a couple components from UITK. They all have text properties defined in their own stylesheet (not inheriting from `body`), so won't be impacted. i.e. UITK components are styled to UITK theme/styles. `body` styles are just an impact for consumer's own components.

This does mean that the intent of this change is less about (specifically) UITK, but more to make Salt components adhere to Salt themes.